src/Mutation.tsx

```diff
@@ -157,7 +161,15 @@ class Mutation<TData = any, TVariables = OperationVariables> extends React.Compo
   };
 
   private mutate = (options: MutationOptions<TVariables>) => {
-    const { mutation, variables, optimisticResponse, refetchQueries, update } = this.props;
+    const { mutation, variables, optimisticResponse, update } = this.props;
+    let refetchQueries = options.refetchQueries || this.props.refetchQueries;
+    if (refetchQueries && refetchQueries.length && Array.isArray(refetchQueries)) {
+      refetchQueries = (refetchQueries as any).map((x: string | PureQueryOptions) => {
+        if (typeof x === 'string' && this.context.operations) return this.context.operations.get(x);
```

Comment:
Will this be removed in the 3.0 as well? Perhaps we can add a not of that here in the code.

Reply:
yep for sure!